lib/src/main/java/uk/gov/di/ipv/cri/address/library/service/AccessTokenService.java

```diff
@@ -143,7 +145,10 @@ public TokenRequest validateTokenRequest(
                             tokenRequest.getClientAuthentication().getClientID().getValue());
             SignedJWT signedJWT = privateKeyJWT.getClientAssertion();
 
-            jwtVerifier.verifyJWT(clientAuthenticationConfig, signedJWT);
+            jwtVerifier.verifyJWT(
+                    clientAuthenticationConfig,
+                    signedJWT,
+                    Arrays.asList(RequiredClaims.EXP.value, RequiredClaims.SUB.value));
```

Comment:
Why use `Array.asList` why not `List.of` same for all the occurences of you've used `Array.asList`

Reply:
Please can you remove ` var configurationServiceLocal = new ConfigurationService();` on line 63 and move `this.configurationService = new ConfigurationService();` to position 63

Reply:
I think you should remove    49 and 58 and also `new ListUtil()` shouldn't be in the constructor it is only used once and should be declared in the method where it is used

Reply:
Addressed all comments